### PR TITLE
Rst cmode

### PR DESCRIPTION
**Pull request by joakim-hove**
More small changes to support Restart.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Comments?

File: src/opm/output/eclipse/AggregateGroupData.cpp [hunk 1]
```diff
@@ -660,8 +661,10 @@ void injectionGroup(const Opm::Schedule&     sched,
                     const std::map<Opm::Group::InjectionCMode, int>& cmodeToNum,
                     IGrpArray&               iGrp)
 {
+    using IGroup = ::Opm::RestartIO::Helpers::VectorItems::IGroup::index;
     const bool is_field = group.name() == "FIELD";
     auto group_parent_list = groupParentSeqIndex(sched, group, simStep);
+    using IGroup = ::Opm::RestartIO::Helpers::VectorItems::IGroup::index;
```
Comment (bska) [inline src/opm/output/eclipse/AggregateGroupData.cpp:667]: Do you need two of these or is this just a merge/rebase conflict resolution?

**Review by bska — COMMENTED** [review]
Looks good to me except for one minor nit.  Unless you intend to make additional changes here, I suggest we merge once the nit is fixed.

File: src/opm/output/eclipse/AggregateGroupData.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/output/eclipse/AggregateGroupData.cpp:667]: Just a rebase - I'll fix

Comment (joakim-hove) [reply src/opm/output/eclipse/AggregateGroupData.cpp:667]: Ehh - they are both needed; in different functions.

**Comment by joakim-hove** [comment]
> Unless you intend to make additional changes here,

There will probably be more changes yes - but the number of moving parts in the restart sequel is so large it feels good to merge the safe changes as I go.

File: src/opm/output/eclipse/AggregateGroupData.cpp (hunk 1 above)
Comment (bska) [reply src/opm/output/eclipse/AggregateGroupData.cpp:667]: > Ehh - they are both needed; in different functions.

I was thinking more about the fact that this function has [two](https://github.com/joakim-hove/opm-common/blob/f9a52791a4429f691f23b67d261a02ae93a2d2cf/src/opm/output/eclipse/AggregateGroupData.cpp#L664) [copies](https://github.com/joakim-hove/opm-common/blob/f9a52791a4429f691f23b67d261a02ae93a2d2cf/src/opm/output/eclipse/AggregateGroupData.cpp#L667) of the same `using IGroup = ...` statement.

**Pull request merged**